muzlag/internal/cmd/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/Drozd0f/gobots/muzlag/internal/cmd/bot"
+)
+
+func main() {
+	session := bot.NewBot()
```

Comment:
Why new bot actually returns not bot with some business logic, but session to the discord server API?